crates/anvil/core/src/eth/transaction/mod.rs

```diff
@@ -87,8 +87,8 @@ pub fn transaction_request_to_typed(
         to,
     ) {
         // legacy transaction
-        (Some(0), _, None, None, None, None, None, None, _) |
```

Comment:
Why are we reformatting the original code? I believe we should maintain the original code format.